config/install/media.type.audio.yml

```diff
@@ -6,7 +6,7 @@ dependencies:
       - islandora_core_feature
 id: audio
 label: Audio
-description: 'A locally hosted audio file.'
+description: 'An audio file, stored in Fedora.'
```

Comment:
I'd recommend not using the words "stored in Fedora" as some might not be using Fedora.

Reply:
Though I see your comment in the comments above about this being defaults.  Just not sure the UI need be so specific here.  Not sure...

Reply:
done!